libs/core/state/src/states/stateable.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { DaffStates } from './state.enum';
+
+/**
+ * The interface an object must implement to consistently describe its current 
+ * state.
+ */
+export interface DaffStateable<T extends DaffStates> {
```

Comment:
Would DaffLoadingState satisfy this generic constraint? I don't think it would.

Reply:
Yes, it does.